examples/latent_sde.py

```diff
@@ -82,16 +82,15 @@ def forward(self, ts, batch_size, eps=None):
         py0 = Normal(loc=self.py0_mean, scale=self.py0_std)
         logqp0 = kl_divergence(qy0, py0).sum(1).mean(0)  # KL(time=0).
 
-        # `trapezoidal_approx` is for SRK. Disabling it gives better performance.
         if args.adjoint:
             zs, logqp = sdeint_adjoint(
                 self, y0, ts, logqp=True, method=args.method, dt=args.dt, adaptive=args.adaptive, rtol=args.rtol,
-                atol=args.atol, options={'trapezoidal_approx': False}
+                atol=args.atol
```

Comment:
Since this super long input arg is removed, could we put atol to the line above to reduce code length?

Reply:
Agreed